sampledb/spreadsheet.xsh

```diff
@@ -0,0 +1,29 @@
+import json
+import pandas as pd
+
+def write_sample_spreadsheet(uids):
+    schema = json.load(open('sample_schema.json'))
+
+    fields = []
+    for name in schema:
+        f = name.replace('_', ' ').title()
+        fields.append(f)
+    fields = {k:[None]*len(uids) for k in fields}
+    fields['Uid'] = list(uids)
+    df = pd.DataFrame.from_records(fields)
+    # Move uid to first column
+    cols = df.columns.tolist()
+    cols.insert(0, cols.pop(cols.index('Uid')))
+    df = df[cols]
+    
+    writer = pd.ExcelWriter('test.xlsx', engine='xlsxwriter')
+    df.to_excel(writer, index=False)
+    sheet = writer.sheets['Sheet1']
+    for i, name in enumerate(df):
+        width = max(len(str(val)) for val in df[name])
+        width = max(width, len(name)) + 1
+        sheet.set_column(i, i, width)
+    writer.save()
+    
+    soffice 'test.xlsx'
```

Comment:
maybe make the file name configurable

Reply:
I thought we would remove the file after the code finishes running, so I'm not sure how useful a configurable file name would be. Though "test.xlsx" probably isn't the best name.

Reply:
We obviously don't want to overwrite and remove another file a user has with the same name, so maybe we can check if the file exists beforehand, and if it does the code can instead use "test-1.xlsx".

Reply:
Do we care about the names? If not just make them the timestamp (they are usually unique, and convey some info).

Reply:
That works.